### PR TITLE
add mbed-os-example-blinky

### PR DESCRIPTION
**Pull request by ghost**
Test purpose

### CODE REVIEW

**Comment by Darkblue38** [comment]
@VeronicaSejeongWoo  
This is a test purpose for mbedTrust.

File: mbed-os-example-blinky.lib [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os-example-blinky/#1747ef46b7466d246ec4cccbfcbd9e1afcaac0c9
```
Comment (weyoui) [inline mbed-os-example-blinky.lib]: Looks good.

Comment (VeronicaSejeongWoo) [reply mbed-os-example-blinky.lib:1]: I can see it. Looks good.

**Comment by ghost** [comment]
This ticket is just showing the PR is working so closed it.

**Pull request closed without merging**